docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: 'Tutorial 10: Account Updates'
+hide_title: true
+sidebar_label: 'Tutorial 10: Account Updates'
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but
+zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+:::note
+
+This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.9.3.
+
+:::
+
+# Tutorial 10: Account Updates
+
+## Permissions, Preconditions, and Composability
+
+## Overview:
+
+Each zkApp transaction constructed in SnarkyJS is composed of [AccountUpdates](../snarkyjs-reference/classes/AccountUpdate), a set of instructions for the Mina network.
+
+Each AccountUpdate can make assertions about its account, apply updates to its account, as well as make assertions about its child AccountUpdates.
+
+Transactions are structured as a list of trees of AccountUpdates - applied with a ["pre-order"](https://en.wikipedia.org/wiki/Tree_traversal) traversal.
+
+Many of the core features of zkApps - permissions, preconditions, composability, and tokens - are each implemented through the use of AccountUpdates. In this tutorial, you learn all of these except tokens and assertions on account update layouts.
```

Comment:
```suggestion
Many of the core features of zkApps—permissions, preconditions, composability, and tokens—are implemented using AccountUpdates. In this tutorial, you learn all of these, except tokens and assertions on account update layouts.
```

Reply:
'layouts' is a new term that is ambiguous. Could we rephrase or remove that word?

Reply:
Update: it's mentioned below, but not prior to this point in the tutorial.